pkgs/applications/misc/buku/default.nix

```diff
@@ -12,20 +11,41 @@ with pythonPackages; buildPythonApplication rec {
     sha256 = "1a33x3197vi5s8rq5fvhy021jdlsc8ww8zc4kysss6r9mvdlk7ax";
   };
 
+  nativeBuildInputs = [
+    pytestcov
+    pytest-catchlog
+    hypothesis
+    pytest
+  ];
+
   propagatedBuildInputs = [
     cryptography
     beautifulsoup4
     requests
     urllib3
   ];
 
-  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
+  preCheck = ''
+    # Fixes two tests for wrong encoding
+    export PYTHONIOENCODING=utf-8
+
+    # Should be upstream, see https://github.com/jarun/Buku/pull/167
+    substituteInPlace setup.py \
+      --replace "hypothesis==3.7.0" "hypothesis>=3.7.0"
```

Comment:
Update: This would get fixed with https://github.com/jarun/Buku/pull/167, thanks @rachmadaniHaryono! Can therefore be removed when Buku is bumped next time